2025-01-20 22:09:27,938 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:09:28,399 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 22:09:29,239 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 22:09:29,240 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:09:29,247 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:09:29,256 INFO: 127.0.0.1 - - [20/Jan/2025 22:09:29] "GET /socket.io/?EIO=4&transport=polling&t=PI5Z8yt HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:09:29,269 INFO: 127.0.0.1 - - [20/Jan/2025 22:09:29] "POST /socket.io/?EIO=4&transport=polling&t=PI5Z90i&sid=LUIcxW--TGp-lt5fAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:09:29,274 INFO: 127.0.0.1 - - [20/Jan/2025 22:09:29] "GET /socket.io/?EIO=4&transport=polling&t=PI5Z90m&sid=LUIcxW--TGp-lt5fAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:10:51,260 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/core/product_manager.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:10:51,683 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:10:52,371 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 22:10:53,910 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 22:10:53,912 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:10:53,920 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:10:53,932 INFO: 127.0.0.1 - - [20/Jan/2025 22:10:53] "GET /socket.io/?EIO=4&transport=polling&t=PI5ZTVM HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:10:53,976 INFO: 127.0.0.1 - - [20/Jan/2025 22:10:53] "POST /socket.io/?EIO=4&transport=polling&t=PI5ZTi2&sid=XfbNtu_G3LYgzHSjAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:10:53,977 INFO: 127.0.0.1 - - [20/Jan/2025 22:10:53] "GET /socket.io/?EIO=4&transport=polling&t=PI5ZTiI&sid=XfbNtu_G3LYgzHSjAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:11,132 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/core/product_manager.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:11,335 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:11,941 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 22:11:12,607 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 22:11:12,608 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:12,616 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:12,975 INFO: 127.0.0.1 - - [20/Jan/2025 22:11:12] "GET /socket.io/?EIO=4&transport=polling&t=PI5ZYL1 HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:12,989 INFO: 127.0.0.1 - - [20/Jan/2025 22:11:12] "POST /socket.io/?EIO=4&transport=polling&t=PI5ZYLJ&sid=Vc8eXCJIA8HdeNtYAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:12,990 INFO: 127.0.0.1 - - [20/Jan/2025 22:11:12] "GET /socket.io/?EIO=4&transport=polling&t=PI5ZYLM&sid=Vc8eXCJIA8HdeNtYAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:13,011 INFO: 127.0.0.1 - - [20/Jan/2025 22:11:13] "GET /socket.io/?EIO=4&transport=polling&t=PI5ZYLf&sid=Vc8eXCJIA8HdeNtYAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:32,929 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/core/product_manager.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:33,305 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:33,845 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 22:11:34,608 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 22:11:34,610 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:34,617 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:34,628 INFO: 127.0.0.1 - - [20/Jan/2025 22:11:34] "GET /socket.io/?EIO=4&transport=polling&t=PI5ZdZ- HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:34,649 INFO: 127.0.0.1 - - [20/Jan/2025 22:11:34] "POST /socket.io/?EIO=4&transport=polling&t=PI5Zddm&sid=QX4wXlzt_drv8-F1AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 22:11:34,653 INFO: 127.0.0.1 - - [20/Jan/2025 22:11:34] "GET /socket.io/?EIO=4&transport=polling&t=PI5Zddp&sid=QX4wXlzt_drv8-F1AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
